tests/GenerateScriptRegressionTests/GenerateScriptRegressionTests.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace GenerateScriptRegressionTests;
+
+public class PackageData
+{
+    public static IEnumerable<object[]> Data => new List<object[]>
+    {
+        new object[] { "Microsoft.CodeAnalysis.CSharp.Workspaces", "4.5.0", PackageType.Reference },
+        new object[] { "System.Threading.Tasks.Extensions", "4.5.4", PackageType.Reference },
+        new object[] { "System.Diagnostics.DiagnosticSource", "7.0.2", PackageType.Reference },
+        new object[] { "Microsoft.CodeAnalysis", "4.5.0", PackageType.Text },
+     };
+}
+
+public enum PackageType
+{
+    Reference,
+    Text
+}
+
+public class GenerateScriptRegressionTests : IClassFixture<GenerateScriptRegressionTestsFixture>, IDisposable
+{
+    private string sandbox_folder = Path.Combine(Environment.CurrentDirectory,"sandbox_folder");
```

Comment:
To be more consistent with the API, I would suggest utilizing `Directory` instead of `Folder`.

Reply:
Changed to sandboxDirecotry